scripts/irods/test/test_iadmin.py

```diff
@@ -1708,6 +1708,138 @@ def test_iadmin_exits_nonzero_on_nonexistent_command__issue_7734(self):
+            )
+
+            self.admin.assert_icommand(["iadmin", "rmuser", test_user_name])
+
+            # Should only see read_object, since that was the perm given to the group
+            _, out, _ = self.admin.assert_icommand(
+                [
+                    "iquest",
+                    "SELECT COLL_ACCESS_USER_ID, COLL_ACCESS_NAME WHERE COLL_NAME = '%s' AND COLL_ACCESS_USER_ID %s"
+                    % (
+                        os.path.join(
+                            self.admin.session_collection, test_collection_name
+                        ),
+                        query_string,
+                    ),
+                ],
+                "STDOUT",
+                ["COLL_ACCESS_NAME = read_object"],
+            )
+            self.assertNotIn("COLL_ACCESS_NAME = modify_object", out)
+
+            # As above, but for data objects
+            _, out, _ = self.admin.assert_icommand(
+                [
+                    "iquest",
+                    "SELECT DATA_ACCESS_USER_ID, DATA_ACCESS_NAME WHERE DATA_NAME = '%s' AND COLL_NAME = '%s' AND DATA_ACCESS_USER_ID %s"
+                    % (test_dataobj_name, self.admin.session_collection, query_string),
+                ],
+                "STDOUT",
+                ["DATA_ACCESS_NAME = read_object"],
+            )
+            self.assertNotIn("DATA_ACCESS_NAME = modify_object", out)
+
+            self.admin.assert_icommand(["iadmin", "rmgroup", test_group_name])
+
+            # Perms should vanish after deletion of associated group
+            self.admin.assert_icommand(
+                [
+                    "iquest",
+                    "SELECT COLL_ACCESS_USER_ID, COLL_ACCESS_NAME WHERE COLL_NAME = '%s' AND COLL_ACCESS_USER_ID %s"
+                    % (
+                        os.path.join(
+                            self.admin.session_collection, test_collection_name
+                        ),
+                        query_string,
+                    ),
+                ],
+                "STDOUT_SINGLELINE",
+                "CAT_NO_ROWS_FOUND: Nothing was found matching your query",
+            )
+
+            # As above, but for data objects
+            self.admin.assert_icommand(
+                [
+                    "iquest",
+                    "SELECT DATA_ACCESS_USER_ID, DATA_ACCESS_NAME WHERE DATA_NAME = '%s' AND COLL_NAME = '%s' AND DATA_ACCESS_USER_ID %s"
+                    % (test_dataobj_name, self.admin.session_collection, query_string),
+                ],
+                "STDOUT_SINGLELINE",
+                "CAT_NO_ROWS_FOUND: Nothing was found matching your query",
+            )
+        finally:
+            self.admin.run_icommand(["iadmin", "rmuser", test_user_name])
+            self.admin.run_icommand(["iadmin", "rmgroup", test_group_name])
+            self.admin.run_icommand(["irmdir", test_collection_name])
+            self.admin.run_icommand(["irm", test_dataobj_name])
```

Comment:
This will move the object to the trash. Let's just delete it outright:
```suggestion
            self.admin.run_icommand(["irm", "-f", test_dataobj_name])
```